assets/app/scripts/app.js

```diff
@@ -294,6 +298,7 @@ angular
   .constant('SOURCE_URL_PATTERN', /^((ftp|http|https|git):\/\/(\w+:{0,1}[^\s@]*@)|git@)?([^\s@]+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$/ )
   // http://stackoverflow.com/questions/9038625/detect-if-device-is-ios
   .constant('IS_IOS', /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream)
+  .constant('DEFAULT_HPA_CPU_TARGET', 80)
```

Comment:
was the plan to make it possible to override this for online? i think we have to put it in constants.js for that